File: cpp/bc1144.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
	int val, sRes, tRes;
	
	cin >> val;
	
	for(int i  = 1; i <= val; i++)
	{
		sRes = pow(i, 2);
		tRes = pow(i, 3);
		
		cout << i << " " << sRes << " " << tRes << endl;
		cout << i << " " << sRes+1 << " " << tRes+1 << endl;
	}	
} 

File: python/bc1144.py
from math import pow

val = int(input())

for c in range(1, val+1):
    sRes = pow(c, 2)
    tRes = pow(c, 3)

    print(f"{c} {sRes:.0f} {tRes:.0f}")
    print(f"{c} {sRes+1:.0f} {tRes+1:.0f}")